src/isIsogram.test.js

```diff
@@ -8,12 +8,22 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
+    expect(typeof isIsogram('word')).toBe('boolean');
+  });
 
+  it(`empty strig is an isogram`, () => {
+    expect(isIsogram(' ')).toBe(true);
   });
 
-  it(`should be case sensitive`, () => {
+  it(`should NOT be case sensitive`, () => {
+    expect(isIsogram('Man')).toBe(true);
+  });
```

Comment:
You don't check case sensitivity in this test, you should check what happens if the same letter is uppercased and lowercase, like `'Bob'`